src/device/usbd.c

```diff
@@ -346,12 +350,59 @@ static bool process_control_request(uint8_t rhport, tusb_control_request_t const
 {
   usbd_control_set_complete_callback(NULL);
 
+  // First pre-compute which driver is in charge of the target (and do bounds checking)
+
+  // FIXME: Should itf and ep be 16-bit since the request's value and index are 16-bit?

```

Comment:
Do we want to also check the high bytes if they are zero? Do we have a function to swap the bytes from LE to native order?